draft-ietf-mls-protocol.md

```diff
@@ -2125,11 +2124,11 @@ the following information for each node in the filtered direct path of the
 sender's leaf, including the root:
 
 * The public key for the node
-* Zero or more encrypted copies of the path secret corresponding to
+* One or more encrypted copies of the path secret corresponding to
   the node
 
-The path secret value for a given node is encrypted for the subtree
-corresponding to the parent's non-updated child, that is, the child
+The path secret value for a given node is encrypted to the subtree
+rooted at the parent's non-updated child. That is, the child
```

Comment:
```suggestion
rooted at the parent's non-updated child, that is, the child
```

If there's a full stop here, the "That is..." part isn't a full sentence.  It might read more smoothly as "i.e."?